ng2-components/ng2-alfresco-core/src/components/pagination/pagination.component.html

```diff
@@ -0,0 +1,49 @@
+<div class="adf-pagination__block adf-pagination__range">
+    <span>{{ 'APP.PAGINATION.SHOW_RANGE' | translate: { range: range.join('-'), total: totalItems } }}</span>
```

Comment:
Translation keys should be present in the `src/i18n/en.json` and probably needs "CORE" namespace for that as that file is currently empty.